R/diet.R

```diff
@@ -109,55 +131,83 @@ getDietMR <- function (params, n = initialN(params),
 #' `r lifecycle::badge("experimental")`
 #' Plots the proportions with which each prey species contributes to the total
 #' biomass consumed by the specified predator species, as a function of the
-#' predator's size. These proportions are obtained with `getDietMR()`.
+#' predator's size. These proportions are obtained with `getDiet()`.
 #'
 #' Prey species that contribute less than 1 permille to the diet are suppressed
 #' in the plot.
 #'
-#' @inheritParams plotSpectra
+#' @param object An object of class \linkS4class{MizerSim} or
+#'   \linkS4class{MizerParams}.
 #' @param species The name of the predator species for which to plot the diet.
+#' @param time_range The time range (either a vector of values, a vector of min
+#'   and max time, or a single value) to average the abundances over. Default is
+#'   the final time step. Ignored when called with a \linkS4class{MizerParams}
+#'   object.
+#' @param wlim A numeric vector of length two providing lower and upper limits
+#'   for the w axis. Use NA to refer to the existing minimum or maximum.
+#' @param return_data A boolean value that determines whether the formatted data
+#' used for the plot is returned instead of the plot itself. Default value is FALSE
+#' @param ... Other arguments (currently unused)
 #'
 #' @return A ggplot2 object, unless `return_data = TRUE`, in which case a data
 #'   frame with the three variables 'w', 'Proportion', 'Prey' is returned.
 #' @export
-#' @seealso [getDietMR()]
+#' @seealso [getDiet()]
 #' @family plotting functions
 
-plotDietMR <- function (object, species = NULL, time_range, wlim = c(1, NA), return_data = FALSE)
+plotDiet <- function (object, species = NULL, time_range, wlim = c(1, NA), return_data = FALSE)
```

Comment:
I have no objections to these additional arguments. However would it make sense to set the defaults for these new arguments so that the default behaviour coincides with that of `mizer::plotDiet()`?